src/umpire/ResourceManager.cpp

```diff
@@ -65,7 +65,6 @@ ResourceManager::getInstance()
 }
 
 ResourceManager::ResourceManager() :
-  m_allocator_names(),
```

Comment:
I'm not sure what ```m_allocator_names``` was used for, but why was it removed?

Reply:
I removed it because it wasn't being used for anything.